### PR TITLE
Test issue creation / label comment automation

### PR DESCRIPTION
**Pull request by crazy-bai**
Whenever an issue is created or a label is marked, the comment will automatically generated.

### CODE REVIEW

File: .github/workflows/comment_automation.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Automated Comment on Issue Creation
+        if: github.event.action == 'opened'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const issueComment = `Hello @${{ github.actor }}, thanks for the issue report! Your report is being reviewed and will be addressed as soon as possible.`;
```
Comment (alisonhofer) [inline .github/workflows/comment_automation.yml:20]: `Hello @${{ github.actor }}, thanks for the issue report! We are reviewing your report and will address it as soon as possible.

**Review by crazy-bai — COMMENTED** [review]
I will change the automated  issue comment to what you comment below.

**Review by chrisjcameron — CHANGES_REQUESTED** [review]
Don't overwrite repo readme

File: .github/workflows/comment_automation.yml
Comment (chrisjcameron) [inline .github/workflows/comment_automation.yml:1]: Are the backticks the correct quotation mark around the issueComment? I don't see any similar examples in YAML.

**Review by chrisjcameron — APPROVED** [review]

**Pull request merged**